esque/messages/avromessage.py

```diff
@@ -122,17 +121,35 @@ def decode_message_from_avro(self, message: Message):
         decoded_message = DecodedAvroMessage(
             decoded_key, decoded_value, message.partition(), key_schema_id, value_schema_id, headers=headers
         )
-        schema_version = it.count(1)
+
         serializable_message = {
             "key": decoded_message.key,
             "value": decoded_message.value,
             "partition": decoded_message.partition,
-            "schema_directory_name": f"{next(schema_version):04}_{key_schema_id}_{value_schema_id}",
+            "key_schema_id": key_schema_id if key_schema_id != -1 else None,
+            "value_schema_id": value_schema_id if value_schema_id != -1 else None,
         }
+
         return key_schema_id, value_schema_id, decoded_message, serializable_message
 
 
 def extract_schema_id(message: bytes) -> int:
     magic_byte, schema_id = struct.unpack(">bI", message[:5])
     assert magic_byte == 0, f"Wrong magic byte ({magic_byte}), no AVRO message."
     return schema_id
+
+
+def get_schemata(base_directory: pathlib.Path, key_schema_id: Optional[int], value_schema_id: Optional[int]):
```

Comment:
Can you annotate the return type?

Reply:
How can I do this with multiple return values? In this case we would have `RecordSchema, RecordSchema`

Reply:
`Tuple(RecordSchema, RecordSchema)`